widget/ui/src/components/Wallet/Wallet.types.ts

```diff
@@ -55,11 +55,11 @@ interface WalletProps {
   onClick: (type: WalletType) => void;
   selected?: boolean;
   description?: string;
+  loadingStatus?: 'failed' | 'success' | 'loading';
```

Comment:
I think `status` is a better name here and also we can use `connected` instead of `sucess`.

Reply:
It's unrelated to the wallet state. it's just for when Meta has not been loaded yet. I've modified both its name and type.